### DISCUSSION TITLE
Unable to build OpenFast using Visual Studio 19 Community version + Intel Fortran Compiler in Windows OS

### DISCUSSION THREAD

**Discussion by kashyap05071994** [OpenFAST/openfast#874] [Q&A]
As per the read me doc of OpenFast, I installed the Windows Studio 2019. Subsequently, Intel Fortran Compiler was installed and integrated with visual studio. Following are the errors obtained during building OpenFast: 

![image](https://user-images.githubusercontent.com/82246846/114254583-e5f02c80-99cd-11eb-8823-4b81d1b2bc90.png)

Please help me resolve the same. I have also read a similar issue on the forum and implemented the same but the errors were sustained(https://github.com/OpenFAST/openfast/issues/322).  
Thank You.

**Comment by bjonkman** [comment]
Which version of Intel Fortran did you install? Did it also install the math kernel library (MKL)? Is this a problem with the `debug` configuration or the `release` configuration?

**Comment by kashyap05071994** [comment]
I was able to successfully build OpenFast. I had not installed the Intel API ToolKit which essentially has the Math Kernel Library. 

Following the build up, I am not sure what is the next step to run the test cases. I am fairly new to this programming language and its fairly difficult to understand the complexity of the document. 

I intend to run the OC4 test case for multiple wind and wave environments for a prescribed number of random numbers and sea states. Is there a way I can automate this entire process using OpenFast. Are there any scripts that illustrate this automation ?

Thank You

**Comment by jjonkman** [comment]
Dear @kashyap05071994,

Regarding scripts to run OpenFAST many times for a loads analysis, please see: https://github.com/OpenFAST/openfast/issues/658.

Best regards,

**Comment by kashyap05071994** [comment]
what is the difference between the debug and the release versions? Which executable shall I use for running the test case 21? Added to these points, i believe the test case files are contained in the r test folder. I am completely lost as to how to integrate the given files in matlab for changing seed numbers for each wind speed and wave height bins. Does openfast run the same way as the fastv8 use to run in the cmd prompt window ?
I really need some help. Thank You.

**Comment by jjonkman** [comment]
Dear @kashyap05071994,

Release mode includes compile-time optimizations that make OpenFAST run much faster.  Debug mode is useful when stepping through the source code through the debugger.  When not debugging, I would run OpenFAST in release mode. 

Yes, OpenFAST can be executed through a Windows command prompt, just like in FAST v8.

Best regards,
